### DISCUSSION TITLE
Stuck in creating reply markup "keyboard button" in laravel

### DISCUSSION THREAD

**Discussion by ferilagi** [westacks/telebot#63] [Q&A] [answered]
```php
$keyboard = new InlineKeyboardMarkup([
    'inline_keyboard' => 'InlineKeyboardButton["text" => "answer1", "callback_data" => "answer1cb"]["text" => "answer2", "callback_data" => "answer2cb"]'
]);
```

and in return 
```php
 return $this->sendMessage([
    'text'  => 'question??',
    'reply_markup'   =>  $keyboard ,
]);
```

and the error  
`WeStacks\TeleBot\Helpers\Type::castMany(): Argument #1 ($values) must be of type iterable, string given, called in D:\sourcecode\laravel-telegrambot\vendor\westacks\telebot\src\Helpers\Type.php on line 33`

i cant find answer in https://westacks.github.io/
can anyone explain proper way / correct way to create inline keyboard in laravel?

**Comment by punyflash** [comment, marked as answer]
To be honest, I don't even understand how you came up with such syntax to create inline keyboard. There are bunch of examples in tests:
https://github.com/westacks/telebot/blob/master/tests/Unit/KeyboardTest.php#L15-L24
https://github.com/westacks/telebot/blob/master/tests/Feature/SendMessageTest.php#L105-L116

Just do this:
```php
 return $this->sendMessage([
    'text'  => 'question??',
    'reply_markup'   =>  [
        'inline_keyboard' => [[
            ['text' => 'answer1', 'callback_data' => 'answer1cb' ],
            ['text' => 'answer2', 'callback_data' => 'answer2cb' ], 
        ]]
    ],
]);
```

**Reply by ferilagi** [reply]
thanks a lot @punyflash
